lib/facade/MonitoringFacade.ts

```diff
@@ -615,4 +616,14 @@ export class MonitoringFacade extends MonitoringScope {
     this.addSegment(segment, props);
     return this;
   }
+
+  /**
+   * Monitor all the resources in the given scope.
```

Comment:
Should we mark this as `@experimental`, or does the current approach feel finalized?

Reply:
Yes I think we can mark it experimental till we feel confident and/or get feedback.